### ISSUE TITLE
<avatarep_suid_[{$similar_thread['uid']}]> not working.

### ISSUE THREAD

**Issue by h2kyaw** [WhiteNeo/lastposter-avatar-mybb18x-series#13]
<avatarep_suid_[{$similar_thread['uid']}]> isn't working in "showthread_similarthreads_bit" template.

![image](https://user-images.githubusercontent.com/4678120/152988346-ec9003cd-1b79-45d8-be93-3b6ef18e8741.png)

image code just phrase as "<avatarep_suid_[1]></avatarep_suid_[1]>" not with img.

Also enabled "Show avatars for similar threads templates into showthread" in Setting.

Version: [Last Poster Avatar](https://community.mybb.com/mods.php?action=view&pid=74) (3.0.4)

**Comment by WhiteNeo** [comment]
Yeah it does not include that part due i will need to add some few variants to the code to allow it, right now i have no idea on that parts due it is been a long time that i have not use mybb anymore.
